src/service/routes_collections.py

```diff
@@ -405,6 +431,41 @@ async def get_match(
     )
 
 
+@ROUTER_SELECTIONS.get(
+    "/selections/",
+    response_model=Selection,
+    summary="Get a selection",
+    description="Get the status and contents of a selection."
+)
+async def get_selection(
+    r: Request,
+    # When I use an alias="" argument in the Header so I can use a sane name for the variable
+    # the server thinks the header is missing
+    KBASE_COLLECTIONS_SELECTION: str = Header(
+        description="The selection ID / token returned when creating a selection."
+    ),
+    verbose: bool = _QUERY_SELECTION_VERBOSE,
+) -> Selection:
+    appstate = app_state.get_app_state(r)
+    hashed_token = _hash_token(KBASE_COLLECTIONS_SELECTION)
+    active_sel = await appstate.arangostorage.get_selection_active(hashed_token)
+    # could save bandwidth by passing verbose to DB layer and not pulling IDs
+    internal_sel = await appstate.arangostorage.get_selection_internal(
```

Comment:
Would it be beneficial to include an option in get_selection_internal that allows the caller to exclude selection_ids from the return?

Reply:
Sure, but I was only planning to do that if it became an issue. Since the changes are all internal to the service it wouldn't break backwards compatibility

I could possibly be convinced otherwise, but if so let's do it in another PR